### PR TITLE
Adding support for async (_acall) for VertexAICommon LLM

### PR DESCRIPTION
**Pull request by pabloem**
# Adding support for async (_acall) for VertexAICommon LLM

This PR implements the `_acall` method under `_VertexAICommon`. Because VertexAI itself does not provide an async interface, I implemented it via a ThreadPoolExecutor that can delegate execution of VertexAI calls to other threads.

Twitter handle: @polecitoem : )

## Who can review?

Community members can review the PR once tests pass. Tag maintainers/contributors who might be interested:

fyi - @agola11 for async functionality
fyi - @Ark-kun from VertexAI

### CODE REVIEW

File: langchain/llms/vertexai.py [hunk 1]
```diff
@@ -91,6 +106,25 @@ def validate_environment(cls, values: Dict) -> Dict:
...
+        Args:
+            prompt: The prompt to pass into the model.
+            stop: A list of stop words (optional).
+            run_manager: A callback manager for async interaction with LLMs.
+
+        Returns:
+            The string generated by the model.
+        """
+        return await asyncio.wrap_future(
+                _VertexAICommon._get_task_executor().submit(self._predict, prompt, stop))
```
Comment (hwchase17) [inline langchain/llms/vertexai.py:126]: is this just `self._get_task_executor()`?

Comment (pabloem) [reply langchain/llms/vertexai.py]: yes correct. I've changed it to match. I've also fixed lint errors.

**Comment by pabloem** [comment]
thanks @hwchase17

**Review by hwchase17 — APPROVED** [review]
lgtm!

**Comment by pabloem** [comment]
fixed 3.10 lint! ready to merge, I think.

**Comment by pabloem** [comment]
sorry @hwchase17 can you please trigger the workflows and merge if they are green?

**Comment by pabloem** [comment]
Ping : ) - @hwchase17

**Comment by pabloem** [comment]
please merge @hwchase17 ?

**Comment by HassanOuda** [comment]
Can you merge this please? @hwchase17 @dev2049

**Comment by hwchase17** [comment]
apologies for delay!

**Pull request merged**